react-app/src/features/forms/waiver/temporary-extension/index.tsx

```diff
@@ -29,11 +30,18 @@ export const TemporaryExtensionForm = () => {
   const { id: waiverId } = useParams<{ id: string }>();
   const { data: submission } = useGetItem(waiverId, { enabled: waiverId !== undefined });
 
+  const [temporaryExtensionType, setTemporaryExtensionType] = useState("1915(b)");
+
   const type =
     submission && submission._source
       ? `${submission._source.authority} ${actionTypeMap[submission._source.actionType]}`
       : null;
 
+  const updateLocalState = (value) => {
```

Comment:
I don't see the value of having a separate function just for updating state, but that's just a nit